ONSdigital/census-rm-ops
# RMC-133 Update action type set

# Motivation and Context
There are new action types, they need to be added to this set so that they can be set up in the UI.

# What has changed
- Update action type set
- Remove DS_Store, add it to gitignore

# How to test?
Build and run with https://github.com/ONSdigital/census-rm-action-scheduler/pull/9
You should be able to set up a fieldwork tranch 2 reminder type (FF2QE) action rule, then see the messages reach the Action.Field queue.

# Links
https://github.com/ONSdigital/census-rm-action-scheduler/pull/9
https://trello.com/c/vhOInkPz/683-rmc-133-distribute-tranche-2-hh-cases-to-fwmtg-13
